drivers/tests/TESTS/mbed_drivers/lp_ticker/main.cpp

```diff
@@ -143,18 +153,18 @@ void test_detach(void)
 {
     LowPowerTicker ticker;
     bool ret;
-    const float ticker_time_s = 0.1f;
-    const uint32_t wait_time_ms = 500;
+    const milliseconds ticker_time = 100ms;
+    const milliseconds wait_time = 500ms;
```

Comment:
Do you even need these two variables ? Can't you just substitute them directly with the chrono times ?

Reply:
Yes that's much neater. Done.